src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFile(fromFileName);
+        String report = generateReport(data);
+        writeToFile(toFileName, report);
+    }
+
+    private String generateReport(String data) {
+        StringBuilder report = new StringBuilder();
+        String[] stringArray = data.split(" ");
+        report.append("supply,").append(Integer.parseInt(stringArray[0]))
+                .append(System.lineSeparator());
+        report.append("buy,").append(Integer.parseInt(stringArray[1]))
+                .append(System.lineSeparator());
+        report.append("result,").append(Integer.parseInt(stringArray[2]))
+                .append(System.lineSeparator());
+        return report.toString();
+    }
+
+    private String readFile(String fileName) {
+        BufferedReader reader = null;
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
```

Comment:
read about try with resources, BufferedReader implements AutoCloseable interface that perfectly matches to try with resources